e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -239,9 +239,20 @@ Scenario('HMCTS admin handles supplementary evidence', async (I, caseListPage, c
   await I.seeCaseInSearchResult(caseId);
 });
 
+Scenario('HMCTS admin search for a given caseId', async (I, caseListPage) => {
+  await I.navigateToCaseList();
+  await caseListPage.searchForCasesWithCaseRef('0000');
+  await I.dontSeeCaseInSearchResult(caseId);
+
+  await I.navigateToCaseList();
+  await caseListPage.searchForCasesWithCaseRef(caseId);
+  await I.seeCaseInSearchResult(caseId);
+});
```

Comment:
we said we would not need to e2e test because we trust this functionality works as expected